src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -442,6 +443,10 @@ const generateTaskRole = (
                   name: 'KUBE_APISERVER_ADDRESS',
                   value: apiserver.uri,
                 },
+                {
+                  name: 'REST_SERVER_URI',
+                  value: process.env.REST_SERVER_URI,
```

Comment:
Please don't use environmental variables here. Put it in `launcherConfig`

Reply:
> Please don't use environmental variables directly here. Put it in `launcherConfig`

fixed, thx